icons/atom.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="1" />
+  <ellipse transform="matrix(0.7071 -0.7071 0.7071 0.7071 -4.9706 12)" cx="12" cy="12" rx="5.4" ry="12" />
+  <ellipse transform="matrix(0.7071 -0.7071 0.7071 0.7071 -4.9706 12)" cx="12" cy="12" rx="12" ry="5.4" />
```

Comment:
While pretty neat, I'm afraid we don't support transforms, so they'll have to be removed. :(
```suggestion
  <path d="M20.2 20.2c2.04-2.03.02-7.36-4.5-11.9-4.54-4.52-9.87-6.54-11.9-4.5-2.04 2.03-.02 7.36 4.5 11.9 4.54 4.52 9.87 6.54 11.9 4.5Z" />
  <path d="M15.7 15.7c4.52-4.54 6.54-9.87 4.5-11.9-2.03-2.04-7.36-.02-11.9 4.5-4.52 4.54-6.54 9.87-4.5 11.9 2.03 2.04 7.36.02 11.9-4.5Z" />
```